pages/dev/gas-receiver.md

```diff
@@ -0,0 +1,134 @@
+# Axelar Gas Receiver
```

Comment:
Not sure where this is going on the outline, but I would suggest it should go beneath General Message Passing: 
# Developers
## Introduction
## Get started
## Transfer tokens
## General Message Passing
## Axelar Gas Receiver